blogWithComments.php

```diff
@@ -0,0 +1,90 @@
+
+<?php
+include ('include/include_all.php');
+
+session_start();
```

Comment:
I'd throw this in `include_all.php`. You always want the session to be initialized.